GVFS/GVFS.UnitTests/Mock/Git/MockGitProcess.cs

```diff
@@ -17,17 +15,33 @@ public MockGitProcess()
             : base(new MockGVFSEnlistment())
         {
             this.CommandsRun = new List<string>();
+            this.StoredCredentials = new Dictionary<string, IList<Credential>>(StringComparer.OrdinalIgnoreCase);
         }
 
         public List<string> CommandsRun { get; }
         public bool ShouldFail { get; set; }
+        public IDictionary<string, IList<Credential>> StoredCredentials { get; }
 
         public void SetExpectedCommandResult(string command, Func<Result> result, bool matchPrefix = false)
         {
             CommandInfo commandInfo = new CommandInfo(command, result, matchPrefix);
             this.expectedCommandInfos.Add(commandInfo);
         }
 
+        public override void StoreCredential(string repoUrl, string username, string password)
+        {
+            IList<Credential> repoCredentials;

```

Comment:
Is there a reason why this is `IList` vs `List`?

Reply:
See my answer to `IDictionary` vs `Dictionary`